### PR TITLE
ADAPT-8299

### PR DESCRIPTION
**Pull request by ethan-lp**
fix for multiple instances of OTI, should not be able to submit until text is entered

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: js/openTextInputModel.js [hunk 1]
```diff
@@ -72,17 +72,8 @@ define([
...
-      let number = allowedChars - countAmount;
-
-      if (number === allowedChars) {
-        return false;
-      }
-
-      if (number < allowedChars) {
-        return true;
-      }
+      const remainingChars = this.get('charactersLeft');
```
Comment (eleanor-heath) [inline js/openTextInputModel.js:75]: you can just do `return this.get('charactersLeft');` instead of creating a variable

**Review by eleanor-heath — APPROVED** [review]

**Review by eleanor-heath — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Pull request merged**